dvc/utils/__init__.py

```diff
@@ -145,18 +147,40 @@ def copyfile(src, dest, no_progress_bar=False, name=None):
         progress.finish_target(name)
 
 
-def move(src, dst):
+def makedirs(path, exist_ok=False, mode=None):
+    if mode:
+        umask = os.umask(0)
+    try:
+        mkdirs(fspath_py35(path), exist_ok=exist_ok, mode=mode or 0o777)
+    finally:
+        if mode:
+            os.umask(umask)
+
+
+def move(src, dst, mode=None):
```

Comment:
maybe renaming this to `atomic_move`? and why not moving this to `fs` aren't this operations related to the file system? (well, they deal with files, maybe my definition of what should be under `fs` is way too relaxed :stuck_out_tongue: )

Reply:
It just replaces the old `move` that we had there.

As to moving to fs, we have a separate ticket for it https://github.com/iterative/dvc/issues/2137 .